### ISSUE TITLE
US 04.01.01

### ISSUE THREAD

**Issue by aeldidi** [CMPUT301W23T12/6ixStacks#7]
As a player, I want a profile with a unique username and my contact information.

# Acceptance Criteria
- The profile page should display the username
- The username should be checked to be unique

# Unknown
- Contact information?

**Comment by aeldidi** [comment]
Whoever is assigned this should keep in mind:

The profile page should be generic so we can pass in a user and get that user's profile page, which will be necessary when implementing the social and friends features.

This means that the edit buttons should only display if the user's secret random number is the same as the one stored on the device.

**Comment by AdibNasreddine** [comment]
Moved firebase integration to another issue.

**Comment by aeldidi** [comment]
blocked by #41 (which should be already done, but not merged at the time this comment was written) and #42.
